package.json

```diff
@@ -490,6 +497,11 @@
           "command": "confluent.schemaViewer.uploadVersion",
           "group": "navigation@3",
           "when": "false"
+        },
+        {
+          "command": "confluent.topic.consume.duplicate",
```

Comment:
Maybe something for a follow-on branch, but what do you think about (also) showing it in other places like the webview/editor tab?

We haven't used it yet, but with the command added under [`editor/title/context`](https://code.visualstudio.com/api/references/contribution-points#contributes.menus) like this:
```
      "editor/title/context": [
        {
          "command": "confluent.topic.consume.duplicate",
          "group": "navigation",
          "when": "activeWebviewPanelId == message-viewer"
        }
      ],
```
It would also show up when right-clicking the tab itself:
<img width="624" alt="image" src="https://github.com/user-attachments/assets/cb394725-05ce-452d-b6f6-19e968a33548">

(But would need some decision on [which group](https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups) it needs to go in, if not its own group.)

Reply:
Easy to do. I'll be doing a follow up PR for a thing in webview panel cache, will update the contributes. Keeping the item at the top can make it visible, as a highly custom command or some sort

Reply:
Addressed in https://github.com/confluentinc/vscode/pull/427/commits/a8e0d1a1b0121371328400e2c863492896aee96a since I had to rebase the branch anyway